allocations/allocations_api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+logger = logging.getLogger('default')
+
+class BALANCE_SERVICE_API:
+
+    def call( self, project_code ):
+        # get token
+        auth = v3.Password(auth_url=settings.OPENSTACK_KEYSTONE_URL,
+                           username=settings.OPENSTACK_SERVICE_USERNAME,
+                           password=settings.OPENSTACK_SERVICE_PASSWORD,
+                           project_id=settings.OPENSTACK_SERVICE_PROJECT_ID,
+                           user_domain_name='default',
+                           user_domain_id='default')
+        sess = session.Session(auth=auth)
+        auth_token = sess.get_token()
+        
+        headers = {'Accept': 'appliaction/json',
```

Comment:
```suggestion
        headers = {'Accept': 'application/json',
```

Sometimes it's nice to take a string that can be misspelled and pull it out to its own constant:

```python
CONTENT_TYPE_JSON = 'application/json'
```

It's possible such a constant already exists in one of the libraries we use.